src/ResourceManager/Compute/documentation/upcoming-breaking-changes.md

```diff
@@ -25,4 +25,38 @@
     https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
 -->
 
-# Upcoming Breaking Changes
\ No newline at end of file
+# Upcoming Breaking Changes
+
+## Release X.0.0 - May 2018
```

Comment:
Do not add this header.  Also, we need to discuss this.  The plan was not to make this a breaking change, but to add an additional parameter set - the user should still be led into the appropriate value.